src/table/__integ__/inline-editing.test.ts

```diff
@@ -150,3 +154,36 @@ test(
     await expect(page.isFocused('[data-testid="focus"]')).resolves.toBe(true);
   })
 );
+
+test(
+  'can activate and dismiss disabled reason popover with mouse',
+  setupTest(async page => {
+    // Click on cell with disabled inline edit
+    await page.click(disabledCell$);
+
+    await expect(page.getText(disabledCellLiveRegion$)).resolves.toContain(
+      "You don't have the necessary permissions to change a BrowserStack origin."
+    );
+
+    // Dismiss with Escape
+    await page.keys(['Escape']);
```

Comment:
when testing in Chrome, if I click on the disabled cell by mouse and then press `Escape` the popover doesn't get dismissed

it works only if I clicked on the lock icon, is that expected?

Reply:
That's not expected. Will take a look

Reply:
@abdhalees I couldnt reproduce this 🤔

Reply:
oh I was looking at the outdated `preview` link and didn't notice that there is a new `dev-pages` link.

all good :)